[IMP] rental: total update of the doc for the rental app (#6780)
odoo/documentation: content/applications/sales/rental.rst

Send quotations, confirm orders, schedule rentals, register products when they are picked up and
returned, and invoice customers from this single platform.

.. seealso::
   - `Odoo Rental: product page <https://www.odoo.com/app/rental>`_
   - `Odoo Tutorials: Rental <https://www.odoo.com/slides/rental-48>`_

.. _rental/pricing:

Dashboard
=========

When the *Rental* application is opened, Odoo reveals the :guilabel:`Rental Orders` dashboard.

.. image:: rental/rental-orders-dashboard.png
   :align: center
   :alt: Example of the Rental Orders dashboard available in the Odoo Rental application.

In the default kanban view, all rentals are visible. Each rental card displays the customer name,
the price of the rental, the related sales order number, along with the status of the rental.

.. note::
   Rental kanban cards that do **not** display a rental status means those rentals have confirmed
   quotations, but haven't been picked up yet.

On the left sidebar, the :guilabel:`Rental Status` for each rental can be found. Beneath that, the
:guilabel:`Invoice Status` of the rentals is accessible. Clicking any option in the left sidebar

> reveals that designated filter option in the dashboard for easier access.

Reviewer: ```suggestion
 filters the displayed rentals on the dashboard.
```

Settings
========

> To access the settings of the Odoo *Rental* application, navigate to :menuselection:`Rental app -->

Reviewer: > To access the settings of the Odoo *Rental* application

While this is concise, I think we can elaborate a little more about why the reader might need extra settings in their rental app. For example, 

```suggestion
To configure additional rental delay costs, availability of rental items, or minimum time of rental,
navigate to :menuselection:`Rental app -->
```

Products`. By default, the :guilabel:`Can be Rented` search filter appears in the search bar.

Each product kanban card displays that product's name, rental price, and product image (if
applicable).

Rental pricing
==============

To adjust the rental pricing on a product, go to the :guilabel:`Products` page in the *Rental* app,
then select the desired product or click :guilabel:`New` to create a new product from scratch.

On the product form, ensure the :guilabel:`Can be Rented` checkbox is ticked. Then, open the
:guilabel:`Rental prices` tab.

.. image:: rental/rental-prices-tab.png
   :align: center
   :alt: How the Settings page appears in the Odoo Rental application.

.. note::
   If creating a rental product outside of the *Rental* app, just ensure the :guilabel:`Can be
   Rented` checkbox is ticked on the product form. By default, this checkbox is ticked whenever a
   product is created directly in the *Rental* application.

Pricing
-------

Under the :guilabel:`Pricing` section of the :guilabel:`Rental prices` tab, designate custom rental
prices and rental periods for the product.

> To add pricing for a rental, click :guilabel:`Add a price`. Then, choose a pricing period (the unit
> of duration of the rental) in the :guilabel:`Period` column, or create a new pricing period by
> typing in the name and clicking :guilabel:`Create`.

Reviewer: This is a good opportunity to use the definition tag! I think we use it like this:
*Term in italics* (:dfn:`definition`)

```suggestion
To add pricing for a rental, click :guilabel:`Add a price`. Then, choose a *pricing period*
(:dfn:`the unit of duration of the rental`) in the :guilabel:`Period` column, or create a new
pricing period by typing in the name and clicking :guilabel:`Create`.
```

   :menuselection:`Rental app --> Configuration --> Settings`, activate :guilabel:`Digital
   Documents`, and click :guilabel:`Save`.

.. note::
   This feature also requires the :doc:`Sign <../finance/sign>` app. If necessary, Odoo installs it
   after activating the :guilabel:`Digital Documents` setting.

To request a customer signature of this nature, select a confirmed rental order, click the
:guilabel:`Sign Documents` button to reveal a :guilabel:`Sign Documents` pop-up window.

.. image:: rental/sign-documents-popup.png
   :align: center
   :alt: The Sign Documents pop-up window that appears in the Odoo Rental application.

From here, select the desired document from the :guilabel:`Document Template` field. Then, click
:guilabel:`Sign Document`. Doing so reveals a :guilabel:`New signature Request` pop-up window.

.. image:: rental/new-signature-request-form.png
   :align: center
   :alt: The New Signature Request pop-up window that appears in the Odoo Rental application.

Upon confirming the information in the :guilabel:`New Signature Request` pop-up form, click
:guilabel:`Sign Now` to initiate the signing process.

> A separate page is then revealed, showcasing the document to be signed. Odoo guides the customer

Author: It's my understanding that it can send it to the customer to sign -- and it would accessible to them via the Customer Portal. I'll add a mention of this to make things a bit more clear.

.. _rental/customer-signature:

Rental orders
=============

> To create a rental order in the *Rental* app, navigate to :menuselection:`Rental app --> Orders -->
> New`. Doing so reveals a blank rental order form to be filled in accordingly.

Reviewer: ```suggestion
To create a rental order in the *Rental* app, navigate to :menuselection:`Rental app --> Orders -->
--> Orders` and clicking :guilabel:`New`. Doing so reveals a blank rental order form to be filled in
accordingly.
```
![image](https://github.com/odoo/documentation/assets/23611711/6580cb04-8040-47b0-aa99-29f833c492b9)

   automatically installs it after activating the :guilabel:`Digital Documents` setting.

To request a customer signature on a rental agreement, select a confirmed rental order, and click
the :guilabel:`Sign Documents` button to reveal a :guilabel:`Sign Documents` pop-up window.

.. image:: rental/sign-documents-popup.png
   :align: center
   :alt: The Sign Documents pop-up window that appears in the Odoo Rental application.

From here, select the desired document from the :guilabel:`Document Template` field. Then, click
:guilabel:`Sign Document`. Doing so reveals a :guilabel:`New Signature Request` pop-up window.

.. image:: rental/new-signature-request-form.png
   :align: center
   :alt: The New Signature Request pop-up window that appears in the Odoo Rental application.

Upon confirming the information in the :guilabel:`New Signature Request` pop-up form, click
:guilabel:`Sign Now` to initiate the signing process.

A separate page is then revealed, showcasing the document to be signed, which is accessible to the
customer via the customer portal.

Odoo guides the customer through the signing process with clear, clickable indicators, and allows

> them to create electronic signatures to quickly complete the form.

Reviewer: How cool!
